team-config.yml

```diff
@@ -673,3 +673,11 @@ da:
     build_notices_channel: "#da-tech-notifications"
   tags:
     application: da-cos-api
+labs:
+  team: "CNP"
+  namespace: "labs"
+  slack:
+    contact_channel: "#platops-labs"
+    build_notices_channel: "#platops-build-notices"
```

Comment:
probably want to create a different channel for this, otherwise people's build failures will spam us,

#labs-build-notices, make sure you create it and invite the jenkins user into it